setup.py

```diff
@@ -43,7 +42,8 @@
     ],
     include_package_data=True,
     install_requires=[
-
+        'django-formtools>=2.0',
```

Comment:
this line tells pip to look in `setup.py`

https://github.com/SexualHealthInnovations/django-wizard-builder/blob/install/requirements/base.txt#L1